src/styles/Common.scss

```diff
@@ -0,0 +1,52 @@
+@import 'Fonts';
+@import 'Constants';
+
+* { box-sizing: border-box; }
+
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.title {
+  height: 2.7rem;
```

Comment:
In this case, as the text is fixed, I think line-height and height are the same, aren't they?